src/Panels/Panels.js

```diff
@@ -1049,7 +1049,9 @@ module.exports = kind(
 		}
 		var orig = ev.originator,
 			idx = this.getPanelIndex(orig),
-			target = this.getSpotlightTarget('LEFT', orig);
+			target = this.getSpotlightTarget('LEFT', orig),
+			active = this.getActive(),
+			h = active ? active.getHeader() : null;
```

Comment:
Not sure we need the ternary here if all we're doing is a truthy check on `h` anyways.